desktop/apps/consignments2/client/reducers.js

```diff
@@ -30,6 +32,20 @@ const stepsMapping = [
 
 const initialState = {
   currentStep: 0,
+  inputs: {
+    authenticity_certificate: 'yes',
+    depth: '',
+    dimensions_metric: 'in',
+    edition: false,
+    height: '',
+    location: '',
+    medium: 'painting',
+    provenance: '',
+    signature: 'yes',
+    title: '',
+    width: '',
+    year: ''
+  },
```

Comment:
Related to https://github.com/artsy/force/pull/1433/files#r116830404. The idea with `<Field />` and overall redux-form architecture is that you can manage so much of it at runtime, without the need to maintain and track independent state. Whats the thinking behind creating this additional tree? For any given form you should be able to retrieve its values for rehydrating app sections, or ensuring (through a config value) that the form field data isn't destroyed or reset after a component is unmounted.